recipesearch/src/Recipes/SearchBar/reducer.js

```diff
@@ -7,7 +7,7 @@ export default function searchBarReducer(state = initialState, action) {
     case 'SEARCH_BAR':
     return{
       ...state,
-      searchResults: [action.payload]
+      searchResults: action.payload
```

Comment:
What's the shape of the payload at this point?

Reply:
The shape of the payload at this point is an array of objects,
Previously it was an array that had an array of objects.